var/spack/repos/builtin/packages/libedit/package.py

```diff
@@ -30,7 +30,6 @@ class Libedit(AutotoolsPackage):
     homepage = "http://thrysoee.dk/editline/"
     url      = "http://thrysoee.dk/editline/libedit-20150325-3.1.tar.gz"
 
-    version('3.1', '43cdb5df3061d78b5e9d59109871b4f6',
-            url="http://thrysoee.dk/editline/libedit-20150325-3.1.tar.gz")
+    version('20150325-3.1', '43cdb5df3061d78b5e9d59109871b4f6')
```

Comment:
Hmm, I could, but what would be the benefit in that? If they added a version `3.1.1-20170213` it would make it difficult to rearrange due to the different number of version characters.